src/styles/index.css

```diff
@@ -0,0 +1,214 @@
+  background-color: #ffbfcb;
+  transform: translateY(-20px);
+  transition: background-color 0.5s, transform 0.5s;
+}
+
+.calendar--start-day-mon .calendar__day:nth-child(1) {
+  margin-left: 0;
+}
+
+.calendar--start-day-tue .calendar__day:nth-child(1) {
+  margin-left: 101px;
+}
+
+.calendar--start-day-wed .calendar__day:nth-child(1) {
+  margin-left: 202px;
+}
+
+.calendar--start-day-thu .calendar__day:nth-child(1) {
+  margin-left: 303px;
+}
+
+.calendar--start-day-fri .calendar__day:nth-child(1) {
+  margin-left: 404px;
+}
+
+.calendar--start-day-sat .calendar__day:nth-child(1) {
+  margin-left: 505px;
+}
+
+.calendar--start-day-sun .calendar__day:nth-child(1) {
+  margin-left: 606px;
+}
+
+.calendar__day:nth-child(1)::before {
+  content: "1";
+}
+
+.calendar__day:nth-child(2)::before {
+  content: "2";
+}
+
+.calendar__day:nth-child(3)::before {
+  content: "3";
+}
+
+.calendar__day:nth-child(4)::before {
+  content: "4";
+}
+
+.calendar__day:nth-child(5)::before {
+  content: "5";
+}
+
+.calendar__day:nth-child(6)::before {
+  content: "6";
+}
+
+.calendar__day:nth-child(7)::before {
+  content: "7";
+}
+
+.calendar__day:nth-child(8)::before {
+  content: "8";
+}
+
+.calendar__day:nth-child(9)::before {
+  content: "9";
+}
+
+.calendar__day:nth-child(10)::before {
+  content: "10";
+}
+
+.calendar__day:nth-child(11)::before {
+  content: "11";
+}
+
+.calendar__day:nth-child(12)::before {
+  content: "12";
+}
+
+.calendar__day:nth-child(13)::before {
+  content: "13";
+}
+
+.calendar__day:nth-child(14)::before {
+  content: "14";
+}
+
+.calendar__day:nth-child(15)::before {
+  content: "15";
+}
+
+.calendar__day:nth-child(16)::before {
+  content: "16";
+}
+
+.calendar__day:nth-child(17)::before {
+  content: "17";
+}
+
+.calendar__day:nth-child(18)::before {
+  content: "18";
+}
+
+.calendar__day:nth-child(19)::before {
+  content: "19";
+}
+
+.calendar__day:nth-child(20)::before {
+  content: "20";
+}
+
+.calendar__day:nth-child(21)::before {
+  content: "21";
+}
+
+.calendar__day:nth-child(22)::before {
+  content: "22";
+}
+
+.calendar__day:nth-child(23)::before {
+  content: "23";
+}
+
+.calendar__day:nth-child(24)::before {
+  content: "24";
+}
+
+.calendar__day:nth-child(25)::before {
+  content: "25";
+}
+
+.calendar__day:nth-child(26)::before {
+  content: "26";
+}
+
+.calendar__day:nth-child(27)::before {
+  content: "27";
+}
+
+.calendar__day:nth-child(28)::before {
+  content: "28";
+}
+
+.calendar__day:nth-child(29)::before {
+  content: "29";
+}
+
+.calendar__day:nth-child(30)::before {
+  content: "30";
+}
+
+.calendar__day:nth-child(31)::before {
+  content: "31";
+}
+
+.calendar--month-length-28 .calendar__day:nth-last-child(-n+3) {
+  display: none;
+}
+
+.calendar--month-length-29 .calendar__day:nth-last-child(-n+2) {
+  display: none;
+}
+
+.calendar--month-length-30 .calendar__day:nth-last-child(-n+1) {
+  display: none;
+}
+
+.calendar--month-length-31 .calendar__day:nth-last-child(-n+0) {
+  display: none;
```

Comment:
The selector '.calendar--month-length-31 .calendar__day:nth-last-child(-n+0)' is incorrect. The nth-last-child pseudo-class with '-n+0' will not select any elements. If you want to ensure that no days are hidden for a 31-day month, you can simply omit this rule or use a more appropriate selector.